app/models/learner-group.js

```diff
@@ -386,4 +386,37 @@ export default DS.Model.extend({
       });
     });
   }),
+
+  /**
+   * Checks if this group or any of its subgroups has any learners.
+   * @property hasLearnersInGroupOrSubgroups
+   * @type {Ember.computed}
+   * @public
+   */
+  hasLearnersInGroupOrSubgroups: computed('users.[]', 'children.@each.hasLearnersInGroupOrSubgroup', function() {
+    return new Promise(resolve => {
+      this.get('users').then(users => {
```

Comment:
This can be checked without loading all of the users like 
```javascript
const userIds = this.hasMany('users').ids();
if (userIds.length) {
  resolve(true);
}
```